### PR TITLE
Feature#20 active storage

### LINKED ISSUE #20: Active Storage Azure
- Azure server configuration to save user's image profile in production environment
- Set environment variables to Heroku server

### PR DESCRIPTION
**Pull request by jnegrete15**
# Azure service

- [x] I added the active storage to save static files in Heroku server using azure Microsoft service.
- [x] I installed **figaro** gem to set local environment variables and set them on heroku as well.
- [x] I fixed login issue on production environment when wrong email or password is entered.
- [x] We've also set styles for the password recovery view.

---
Screenshot from 2021-05-19 15-41-53

### CODE REVIEW

**Review by LPMendoza — APPROVED** [review]
Good job man, I like the fact that you used Azure to storage static files

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore application configuration
+/config/application.yml
```
Comment (OchoaHub) [inline .gitignore:40]: Nice

File: Gemfile [hunk 2]
```diff
@@ -61,4 +61,6 @@ gem 'rubycritic', require: false
 gem 'sandi_meter', '~> 1.2'
 gem 'wkhtmltopdf-binary'
 gem 'wicked_pdf', git: 'https://github.com/mileszs/wicked_pdf.git', branch: 'master'
+gem 'azure-storage-blob'
```
Comment (OchoaHub) [inline Gemfile:64]: Azure!? OMG Nice

**Review by OchoaHub — APPROVED** [review]
LGTM

**Pull request merged**